app/views/govuk_publishing_components/components/_document_list.html.erb

```diff
@@ -14,7 +14,10 @@
     <ol class="gem-c-document-list<%= margin_bottom_class %><%= margin_top_class %>">
   <% end %>
     <% items.each do |item| %>
-      <li class="gem-c-document-list__item<%= within_multitype_list_class %> <%= brand_helper.brand_class %>">
+      <li class="gem-c-document-list__item<%= within_multitype_list_class %> <%= brand_helper.brand_class %> <%= 'gem-c-document-list__item--highlight' if item[:highlight] %>">
```

Comment:
Should we follow the pattern we use for `within_multitype_list_class` and construct the class at the top of the file to avoid too much logic within the HTML itself?

Reply:
Yes, have changed now, thanks.